Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabbedCommandBar/TabbedCommandBar.bind

```diff
@@ -19,7 +19,7 @@
                     <AppBarButton Icon="Paste" Label="Paste"/>
                     <AppBarSeparator />
                     <AppBarElementContainer>
-                        <ComboBox SelectedIndex="0">
+                        <ComboBox SelectedIndex="0" Width="175">
```

Comment:
eww

Reply:
Yeah, I don't like it either, but without it the ComboBox changes width depending on how long the selected text is.

Reply:
This is just a sample page right? So the hardcoded value seems fine

Reply:
What about a MinWidth, or a MaxWidth?

Reply:
I'll make it use `MinWidth` instead. `MaxWidth` isn't what I'm going for, since it would still resize if the text is shorter than the longest string in the menu.